bin/mkmf.template.nci

```diff
@@ -38,14 +38,15 @@ endif
 
 FPPFLAGS := -fpp -Wp,-w $(INCLUDE)
 FFLAGS := -fno-alias -safe-cray-ptr -fpe0 -ftz -assume byterecl -i4 -r8 -traceback -nowarn -check noarg_temp_created -assume nobuffered_io -convert big_endian -grecord-gcc-switches -align all
-FFLAGS_OPT = -O2 -debug minimal -xHost
-FFLAGS_DEBUG = -g -O0 -debug all -check -check noarg_temp_created -check nopointer -warn -warn noerrors -ftrapuv
-FFLAGS_REPRO = -fp-model precise
-FFLAGS_VERBOSE = -v -V -what
+FFLAGS_OPT := -g3 -O2 -axCORE-AVX2 -debug all -check none -qopt-report=5 -qopt-report-annotate
+FFLAGS_DEBUG := -g3 -O0 -debug all -check -check noarg_temp_created -check nopointer -warn -warn noerrors -ftrapuv
+FFLAGS_REPRO := -fp-model consistent -fp-model source -align all
```

Comment:
why the change from `-fp-model precise` to `-fp-model consistent`? `consistent` is probably slower. Is it needed for reproducible results between Broadwell and Cascade Lake?

Reply:
Well spotted. I will change to precise and re-test.

Reply:
Addressed by commit 36ed508